cmd/spec/example_reservation.go

```diff
@@ -63,6 +64,13 @@ var GenericReservationResponsePayloadListExample = payloads.GenericReservationLi
 		&GenericReservationResponsePayloadSuccessExample,
 		&GenericReservationResponsePayloadFailureExample,
 	},
+	Metadata: page.Metadata{
+		Total: 2,
+	},
+	Links: page.Links{
+		Previous: "/api/provisioning/v1/reservations?limit=100&offset=0",
```

Comment:
Isn't this (the result in example the first page, so the Previous should be alse empty, right?

Also I see total 2, but there are three results (Pending, Success, Failure) :)

Reply:
Yes, copy-past mistake :) fixing